### DISCUSSION TITLE
Display only working after firmware upload

### DISCUSSION THREAD

**Discussion by sergio-daniels** [ElectroTechnique/electrotechnique.github.io#78] [Hardware Issues]
Hi I just finished building my Tsynth. First try everything was working except that 1/4 of the display at the right was cut off, so I thought the display was broken. I exchanged the display and now it is working fine but if I power of and on the display lights up but does not show any thing. If I press the reset button several time sometimes it shows the screen with the version correctly but after that it goes dark again. Often If I upload the firmware with Teensduino the display is working correctly until next reboot.
Sometimes when the display is working touching the rotary nob or close by the display goes black again. I checked all the solder and it looks good to me. The PCB is version 1.3
I also have a noise from ground loop when connected to a computer.
Any guess what could keep the display dark when just plugging it in?
Kind regards Sergio

**Comment by Electrotechnique** [comment]
[Use a USB isolator to cure the ground loop](https://www.aliexpress.com/item/1005002946734805.html?spm=a2g0o.productlist.0.0.e59a16edvNfVsi&algo_pvid=0bdcbb04-a5f2-4c78-81ef-d88b102be26b&aem_p4p_detail=20220128000622178801883870100010515139&algo_exp_id=0bdcbb04-a5f2-4c78-81ef-d88b102be26b-0&pdp_ext_f=%7B%22sku_id%22%3A%2212000022911257870%22%7D&pdp_pi=-1%3B5.23%3B-1%3B208%40salePrice%3BGBP%3Bsearch-mainSearch)
Or use a USB mains adapter or a powerbank.

The display problem is strange. Can you provide high resolution pictures of both sides of the PCB please and some information on what and where the display was purchased from. Send them to info@electrotechnique.cc

**Comment by sergio-daniels** [comment]
Thanks for your help I sent you an email with the images.

**Comment by sergio-daniels** [comment]
Me again. I finally got it to work. Sorry to bother you. I must have broken the wire from the Teensy display RST wire to the display RST solder point on the PCB. Now it works perfectly. Thanks again.

**Comment by Electrotechnique** [comment]
Great. It's usually a hardware (soldering) issue. That's useful to know but a bit mysterious still. I would have thought there would be no screen updates if there was no reset. Hope you enjoy the sound!

**Comment by sergio-daniels** [comment]
Yes, It puzzled me as well because it was working from time to time. I love
the sound especially at the moment through USB audio since there is no
ground loop. It's a great design and very rich in functions. Also very nice
to have the source code and able to look inside and to learn how its
implemented.
Great device. Thanks a lot.
Sergio

Am Sa., 29. Jan. 2022 um 02:56 Uhr schrieb ElectroTechnique <
***@***.***>:

> Great. It's usually a hardware (soldering) issue. That's useful to know
> but a bit mysterious still. I would have thought there would be no screen
> updates if there was no reset. Hope you enjoy the sound!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ElectroTechnique/electrotechnique.github.io/discussions/78#discussioncomment-2070272>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACDHH2BRCUCJPZ2QFA4BZBDUYNCNLANCNFSM5M7JORWQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> <ElectroTechnique/electrotechnique.github.
> ***@***.***>
>


-- 
Gruss
Sergio
